declarejs.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * This script will generate exports from the js components. The goal is to rollout gradually typescript versions
```

Comment:
Could you please move this into a `scripts` directory?

Reply:
sure